proto/minder/v1/minder.proto

```diff
@@ -2232,10 +2431,22 @@ message RuleType {
     Definition def = 4;
 
     // description is the description of the rule type.
-    string description = 5;
+    string description = 5 [
+        (buf.validate.field).string = {
+            pattern: "[A-Za-z][-/.!?,:;[:word:] *#\\[\\]()`{}]*",
```

Comment:
Again, this is markdown, so we can do this for now, but we may need to parse in Go code.